packages/nx/src/utils/package-manager.ts

```diff
@@ -142,6 +149,23 @@ export function getPackageManagerCommand(
         list: 'npm ls',
       };
     },
+    bun: () => {
+      // TODO: Remove this
+      process.env.npm_config_legacy_peer_deps ??= 'true';
```

Comment:
Why are you adding something with a TODO of remove this without an explanation of the conditions for its removal?

Reply:
Just copied from the NPM version